src/test/common/CMakeLists.txt

```diff
@@ -357,10 +357,17 @@ add_executable(unittest_async_completion test_async_completion.cc)
 add_ceph_unittest(unittest_async_completion)
 target_link_libraries(unittest_async_completion ceph-common Boost::system)
 
+
 add_executable(unittest_async_max_concurrent_for_each test_async_max_concurrent_for_each.cc)
 add_ceph_unittest(unittest_async_max_concurrent_for_each)
 target_link_libraries(unittest_async_max_concurrent_for_each ceph-common Boost::system Boost::context)
 
+if(NOT WIN32)
+add_executable(unittest_async_co_throttle test_async_co_throttle.cc)
```

Comment:
I'm a bit concerned that we may hit the seg faults again later.

https://paste.openstack.org/raw/bLnxLRTrzYjCyREDWQS8/

Reply:
@cbodley Could you take a look at this?

Reply:
how do we prevent windows from running this test?

Reply:
This patch already skips the test. My concern was that we may see the segmentation faults on Windows if this async coroutine throttling code gets used by librados, librbd or libcephfs.

I'm wondering if it's caused by the llvm version that we're using.